api/src/opentrons/protocol_api/module_contexts.py

```diff
@@ -271,60 +271,59 @@ def run(ctx):
 
     """
 
-    # TODO(mc, 2022-02-05): this type annotation is misleading;
-    # a SynchronousAdapter wrapper is actually passed in
-    _module: modules.tempdeck.TempDeck  # type: ignore[assignment]
+    _core: AbstractTemperatureModuleCore[AbstractLabware[AbstractWellCore]]
 
     @publish(command=cmds.tempdeck_set_temp)
     @requires_version(2, 0)
     def set_temperature(self, celsius: float) -> None:
-        """Set the target temperature, in C.
+        """Set the target temperature, in °C, waiting for the target to be hit.
 
-        Must be between 4 and 95C based on Opentrons QA.
+        Must be between 4 and 95°C based on Opentrons QA.
```

Comment:
```suggestion
        Must be between 4 and 95 °C based on Opentrons QA.
```
SI space between number and units